### ISSUE TITLE
Animation Api example shows dependency error in expo playground

### ISSUE THREAD

**Issue by royalbhati** [facebook/react-native-website#1878]
https://reactnative.dev/docs/animations

```js
 React.useEffect(() => {
    Animated.timing(
      fadeAnim,
      {
        toValue: 1,
        duration: 10000,
      }
    ).start();
  }, [])


```
The above code is missing a dependency in Array supplied to useEffect 

**FIX** :  add fadeAnim to dependency array

**Comment by Simek** [comment]
Can you elaborate on this problem? In the current example `fadeAnim` is defined (line 5 of example) as a reference using `useRef` hook which is based on the `Animated.Value()` current value.

**Comment by anthonyhumphreys** [comment]
Was about to post this myself - currently the example shows a dependency error, eslint expects to see `fadeAnim` in the deps array.

```useEffect(() => {
    Animated.timing(fadeAnim, {
      toValue: 1,
      duration: 1000,
    }).start();
  }, [fadeAnim]);

I'm not sure if that is correct though, or if eslint is in the wrong here, should that be a dep?

**Comment by anthonyhumphreys** [comment]
Looks like this was fixed in #1879
